website/src/react-native/showcase.js

```diff
@@ -5,1128 +5,264 @@
+    infoTitle: 'Using React Native: One Year Later',
   },
   {
     name: 'Discovery VR',
     icon: 'http://a2.mzstatic.com/us/r30/Purple6/v4/d1/d5/f4/d1d5f437-9f6b-b5aa-5fe7-47bd19f934bf/icon175x175.png',
     linkAppStore: 'https://itunes.apple.com/us/app/discovery-vr/id1030815031?mt=8',
     linkPlayStore: 'https://play.google.com/store/apps/details?id=com.discovery.DiscoveryVR',
     author: 'Discovery Communications',
-    blog: [
-      "https://medium.com/ios-os-x-development/an-ios-developer-on-react-native-1f24786c29f0",
-    ],
-  },
-  {
-    name: 'Exponent',
-    icon: 'http://a1.mzstatic.com/us/r30/Purple20/v4/b0/26/72/b02672b5-d347-6df1-e20f-de5f5c039664/icon175x175.jpeg',
-    linkAppStore: 'https://itunes.apple.com/us/app/exponent/id982107779?mt=8',
-    linkPlayStore: 'https://play.google.com/store/apps/details?id=host.exp.exponent',
-    author: 'Exponent',
-  },
-  {
-    name: 'Helium',
-    icon: 'http://a1.mzstatic.com/us/r30/Purple49/v4/df/a6/22/dfa622c4-e29c-c720-f32e-3fec99d14337/icon175x175.png',
-    link:  'https://itunes.apple.com/app/id1071378510',
-    author: 'Helium',
-  },
-  {
-    name: 'li.st',
-    icon: 'https://lh3.googleusercontent.com/tXt0HgJ7dCgOnuQ-lQr1P7E57mnOYfwXhRsV9lGcPwHPVvrDAN6YmpLVFgy88qKrkFI=w300-rw',
-    link: 'https://play.google.com/store/apps/details?id=st.li.listapp',
-    author: 'li.st + Exponent',
-  },
-  {
-    name: 'Lrn',
-    icon: 'http://is4.mzstatic.com/image/pf/us/r30/Purple1/v4/41/a9/e9/41a9e9b6-7801-aef7-2400-2eca14923321/mzl.adyswxad.png',
-    link: 'https://itunes.apple.com/us/app/lrn-learn-to-code-at-your/id1019622677',
-    author: 'Lrn Labs, Inc',
+    infoLink: "https://medium.com/ios-os-x-development/an-ios-developer-on-react-native-1f24786c29f0",
+    infoTitle: '"I may never write an iOS app in Objective-C or Swift again."',
   },
   {
     name: 'Movie Trailers by MovieLaLa',
     icon: 'https://lh3.googleusercontent.com/16aug4m_6tvJB7QZden9w1SOMqpZgNp7rHqDhltZNvofw1a4V_ojGGXUMPGiK0dDCqzL=w300',
     linkAppStore: 'https://itunes.apple.com/us/app/movie-trailers-by-movielala/id1001416601?mt=8',
     linkPlayStore: 'https://play.google.com/store/apps/details?id=com.movielala.trailers',
-    author: 'MovieLaLa'
+    infoLink: 'http://variety.com/2016/digital/news/movielala-1-4-million-seed-round-hollywood-angel-investors-1201678139/',
+    infoTitle: 'MovieLaLa Closes $1.4 Million Seed Round',
   },
   {
     name: 'Myntra',
     icon: 'http://a5.mzstatic.com/us/r30/Purple6/v4/9c/78/df/9c78dfa6-0061-1af2-5026-3e1d5a073c94/icon350x350.png',
-    link: 'https://itunes.apple.com/in/app/myntra-fashion-shopping-app/id907394059',
-    author: 'Myntra Designs',
-  },
-  {
-    name: 'Noodler',
-    icon: 'http://a5.mzstatic.com/us/r30/Purple6/v4/d9/9a/69/d99a6919-7f11-35ad-76ea-f1741643d875/icon175x175.png',
-    link: 'http://www.noodler-app.com/',
-    author: 'Michele Humes & Joshua Sierles',
-  },
-  {
-    name: 'React Native Playground',
```

Comment:
Since this is more of a "react native resource" and less of a "app that proves you can use react native in the real world", I think we should move this out of the "showcase" but we can put it in the "more resources" tab once we build that.